code/modules/admin/watchlist.dm

```diff
@@ -36,6 +36,8 @@
 	if(browse)
 		watchlist_show(target_sql_ckey)
 
+	add_note(target_ckey, "Added to Watchlist - [reason]", null, usr.ckey, 0, null, 0)
```

Comment:
Depends why they're on it. "On probation for greytiding" shouldn't be a secret but I know admins like to keep metagaming suspicions to themselves

I personally don't think secretly marking a player as awful to the point that every admin must be reminded every time they log in without ever letting them explain their actions or defend themselves is good though